vue3/src/style.css

```diff
@@ -0,0 +1,17 @@
+:root {
+  font: var(--gcds-font-text);
```

Comment:
Sorry, one more small thing I forgot to add yesterday. Can we also add a media query to adjust the font-size for mobile, please?

```
@media only screen and (width < 48em) {
      font: var(--gcds-font-text-mobile);
}
```